.azure-pipelines/run-test-template.yml

```diff
@@ -56,9 +56,11 @@ steps:
 
 - script: |
     pwd
+    git_user_name=$(USER_NAME)
+    git_api_token=$(API_TOKEN)
```

Comment:
Secret variables should be passed in through environment variable. Please refer to:

https://docs.microsoft.com/en-us/azure/devops/pipelines/process/set-secret-variables?view=azure-devops&tabs=yaml%2Cbash#use-a-secret-variable-in-the-ui